packages/kbn-securitysolution-autocomplete/src/field/use_field.tsx

```diff
@@ -32,7 +32,7 @@ const getExistingFields = (indexPattern: DataViewBase | undefined): DataViewFiel
 };
 
 const getSelectedFields = (selectedField: DataViewField | undefined): DataViewFieldBase[] => {
-  return selectedField ? [selectedField] : [];
+  return selectedField && selectedField.name.trim() !== '' ? [selectedField] : [];
```

Comment:
Nit: can't we use the `paramContainsSpace` for checking the `electedField && selectedField.name.trim() !== ''` so that we can unify the condition in different places ?

Reply:
Updated!